tests/Microsoft.Identity.Test.Integration/Infrastructure/SeleniumExtensions.cs

```diff
@@ -48,7 +48,7 @@ public static IWebDriver CreateDefaultWebDriver()
 
             // ~2x faster, no visual rendering
             // remove when debugging to see the UI automation
-            options.AddArguments("headless");
+            //options.AddArguments("headless");
```

Comment:
remove dead code?

Reply:
or add the code back in?

---
In reply to: [262183473](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/937#discussion_r262183473) [](ancestors = 262183473)